EngineLayer/Ms2ScanWithSpecificMass.cs

```diff
@@ -19,9 +19,12 @@ public Ms2ScanWithSpecificMass(MsDataScan mzLibScan, double precursorMonoisotopi
 
             TheScan = mzLibScan;
 
-            ExperimentalFragments = neutralExperimentalFragments ?? GetNeutralExperimentalFragments(mzLibScan, commonParam);
+            if (commonParam.DissociationType != DissociationType.LowCID)
```

Comment:
this is the main thing i changed. the old code had a bug in it where the MS2 scan would get deconvoluted for lowCID where it should have skipped deconvolution